src/controllers/dynakube/dtclient_builder.go

```diff
@@ -34,8 +35,7 @@ type DynatraceClientProxy struct {
 }
 
 const (
-	proxy        = "proxy"
-	certificates = "certs"
+	proxy = "proxy"
```

Comment:
wouldn't these const fit better in the `dtclient` package ?

Reply:
moved